src/views/Item/Item.js

```diff
@@ -29,17 +30,22 @@ function Item(props) {
     )
   }
 
+  let contentShorthandValue = content
+  if (
+    contentShorthandValue === undefined &&
+    [description, extra, header, meta].some((prop) => !_.isNil(prop))
+  ) {
+    contentShorthandValue = {}
+  }
+
   return (
     <ElementType {...rest} className={classes}>
       {ItemImage.create(image, { autoGenerateKey: false })}
 
-      <ItemContent
-        content={content}
-        description={description}
-        extra={extra}
-        header={header}
-        meta={meta}
-      />
+      {ItemContent.create(contentShorthandValue, {
+        autoGenerateKey: false,
+        overrideProps: { description, extra, header, meta },
```

Comment:
Alternatively you could use `defaultProps` here, but I think it makes sense for them to override because they're more 'specific'.